javadoc/allclasses-frame.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!-- NewPage -->
```

Comment:
add `javadoc` to your .gitignore and make sure these files aren't tracked by git